pkg/apis/messaging/v1alpha1/subscription_lifecycle.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // subCondSet is a condition set with Ready as the happy condition and
 // ReferencesResolved and ChannelReady as the dependent conditions.
-var subCondSet = apis.NewLivingConditionSet(SubscriptionConditionReferencesResolved, SubscriptionConditionAddedToChannel, SubscriptionConditionChannelReady)
+var SubCondSet = apis.NewLivingConditionSet(SubscriptionConditionReferencesResolved, SubscriptionConditionAddedToChannel, SubscriptionConditionChannelReady)
```

Comment:
do you want to export this?

Reply:
For this https://github.com/knative/eventing/blob/ab85ce817d776d8ba59b161b18b25930c19c48be/pkg/apis/eventing/v1alpha1/trigger_lifecycle.go#L92, I will need SubCondSet to get top level condition. Is there any disadvantage to export this?

Reply:
something could edit it and do unexpected things